README.md

```diff
@@ -1 +1,915 @@
+  - [Installation](#installation)
+    - [Linux / macOS](#linux--macos)
+  - [Before you start](#before-you-start)
+  - [Global flags](#global-flags)
+  - [Commands](#commands)
+    - [The `add` command](#the-add-command)
+    - [The `start` command](#the-start-command)
+    - [The `stop` command](#the-stop-command)
+    - [The `delete` command](#the-delete-command)
+    - [The `list` command](#the-list-command)
+    - [The `status` command](#the-status-command)
+    - [The `config use-deployment` command](#the-config-use-deployment-command)
+    - [The `connect rasa` command](#the-connect-rasa-command)
+    - [The `auth login` command](#the-auth-login-command)
+    - [The `auth logout` command](#the-auth-logout-command)
+  - [Model Management Commands](#model-management-commands)
+    - [The `model delete` command](#the-model-delete-command)
+    - [The `model download` command](#the-model-download-command)
+    - [The `model list` command](#the-model-list-command)
+    - [The `model tag` command](#the-model-tag-command)
+    - [The `model upload` command](#the-model-upload-command)
+  - [Examples of usage](#examples-of-usage)
+    - [Run Rasa X / Enterprise with a local Rasa Server](#run-rasa-x--enterprise-with-a-local-rasa-server)
+    - [Run Rasa X / Enterprise with mounted a local Rasa project](#run-rasa-x--enterprise-with-mounted-a-local-rasa-project)
+    - [Upgrade Rasa X / Enterprise version](#upgrade-rasa-x--enterprise-version)
+    - [Deploy Rasa X in one of the public cloud providers](#deploy-rasa-x-in-one-of-the-public-cloud-providers)
+    - [Upload a model to Rasa X](#upload-a-model-to-rasa-x)
+  - [Development](#development)
+    - [How to run it?](#how-to-run-it)
+    - [Kind cluster for developing purposes](#kind-cluster-for-developing-purposes)
+    - [Deploy Rasa X with mounted a local path](#deploy-rasa-x-with-mounted-a-local-path)
+    - [Open Rasa X in a web browser](#open-rasa-x-in-a-web-browser)
+    - [Deploy Rasa X with mounted a local path and a custom Docker image](#deploy-rasa-x-with-mounted-a-local-path-and-a-custom-docker-image)
+  - [License](#license)
+
+## Prerequisites
+
+- Kubernetes cluster >= 1.14+
+
+   or
+
+- kind (for local mode)
+
+## Installation
+
+### Linux / macOS
+
+- Binary downloads of `rasactl` can be found on [the Releases page](https://github.com/rasahq/rasactl/releases/latest).
+
+```text
+$ curl -L https://github.com/RasaHQ/rasactl/releases/download/0.0.7/rasactl_0.0.7_darwin_amd64.tar.gz
+$ tar -zxvf rasactl_0.0.7_darwin_amd64.tar.gz
+$ cp rasactl_0.0.7_darwin_amd64/rasactl /usr/local/bin/
+```
+
+- Using `brew`
+
+```text
+$ brew tap rasahq/rasactl
+$ brew install rasactl
+```
+
+## Before you start
+
+Below you can find several things that are good to know and keep in mind when you use `rasactl`.
+
+- `rasactl` executes an operation always only for a given deployment. The order in which `rasactl` determine a deployment to use:
+
+  1. A deployment name passed as an argument.
```

Comment:
Which argument?

Reply:
as an argument passed in the CLI, I added more description to make it clear.

Reply:
thanks! i figured it out later by seeing the Usage sections, but thought it could be clearer up here